Sources/SwiftDocC/Model/Rendering/DocumentationContentRenderer.swift

```diff
@@ -211,11 +211,16 @@ public class DocumentationContentRenderer {
         guard let symbol = node.semantic as? Symbol,
               let currentPlatforms = documentationContext.configuration.externalMetadata.currentPlatforms,
               !currentPlatforms.isEmpty,
-              let symbolAvailability = symbol.availability
+              let symbolAvailability = symbol.availability?.availability
         else { return false }
+        
+        // If the symbol does not have any availability item it can't be `beta`.
+        guard !symbolAvailability.isEmpty else {
+            return false
+        }
```

Comment:
FYI: Since both guard-statements return `false` and the first guard-statement already has many conditions you can move this new condition into the same guard-statement as the others.

```suggestion
              let symbolAvailability = symbol.availability?.availability
              !symbolAvailability.isEmpty // A symbol without availability items can't be in beta.
        else { return false }
```